modules/ept-frontend/client/components/profile/profile.directive.js

```diff
@@ -228,61 +229,55 @@ function(Conversations, Messages, User, Session, Alert, PreferencesSvc, $filter,
       // Create Conversation
       this.newConversation = {};
       this.showConvoModal = false;
+
```

Comment:
you should not include arbitrary newline additions in your commits with code changes